src/page/AddingEvent/index.tsx

```diff
@@ -0,0 +1,293 @@
+          setPreImages((prev) => [...prev, reader.result]); // 이미지 미리보기 구현
+        };
+      }
+    }
+  };
+
+  const postEvent = () => {
+    const startDate = `${eventInfo.start_date.year}-${eventInfo.start_date.month}-${eventInfo.start_date.date}`;
+    const endDate = `${eventInfo.end_date.year}-${eventInfo.end_date.month}-${eventInfo.end_date.date}`;
+
+    if (imageList.file) {
+      for (let i = 0; i < imageList.file.length; i += 1) {
+        const url = imageList.presigned[i].pre_signed_url;
+        uploadImage({ presignedUrl: url, file: imageList.file[i] });
+      }
+    }
+
+    const requestData = {
+      title: eventInfo.title,
+      content: editor,
+      start_date: startDate,
+      end_date: endDate,
+      thumbnail_image: imageList.presigned.map((img) => img.file_url),
+    };
+
+    addEvent(requestData);
+
+    return requestData;
+  };
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.event}>
+        <div>
+          <p className={styles.event__paragraph}>사진</p>
+          <div className={styles.event__divide}>
+            <small className={styles.event__count}>이벤트/공지와 관련된 사진을 올려보세요.</small>
+            <small className={styles.event__count}>
+              {preImages.length}
+              /3
+            </small>
+          </div>
+          {preImages.length > 0 && (
+            <div className={styles.event__images}>
+              {
+                preImages.map((src, id) => {
+                  if (typeof src === 'string') {
+                    return (
+                      <div className={styles['event__each-image']}>
+                        <img src={src} alt="미리보기" className={styles.event__image} />
+                        <button
+                          type="button"
+                          className={styles['event__each-image--delete']}
+                          onClick={() => {
+                            const deleted = preImages.filter((item, idx) => idx !== id);
+                            setPreImages([...deleted]);
+                          }}
+                        >
+                          <Delete />
+                        </button>
+                      </div>
+                    );
+                  }
+                  return '';
+                })
+              }
+            </div>
+          )}
+          <label htmlFor="fileUpload" className={styles.event__upload}>
+            사진 등록하기
+          </label>
+          <input
+            type="file"
+            id="fileUpload"
+            placeholder="사진 등록하기"
+            accept="image/*"
+            multiple
+            maxLength={3}
+            onChange={handleImages}
+            style={{ display: 'none' }}
+          />
+        </div>
+        <div>
+          <div className={styles.event__divide}>
+            <p className={styles.event__paragraph}>제목</p>
+            <small className={styles.event__count}>
+              {eventInfo.title.length}
+              /25
```

Comment:
소소한 한줄습관입니다 ㅋㅋ 요렇게쓰면 띄어쓰기도 보기 쉬운 것 같아용
```suggestion
              {`${eventInfo.title.length}/25`}
```

Reply:
넵 가독성이 향상됐네요